packages/blade/src/components/Icons/Icons.stories.tsx

```diff
@@ -114,36 +115,50 @@ Icon.args = {
   icon: 'CreditCardIcon',
 };
 
+const IconList = ({
+  IconComponent,
+  iconName,
+  ...args
+}: {
+  IconComponent: IconComponent;
+  iconName: keyof typeof iconMap;
+} & IconProps): JSX.Element => {
+  return (
+    <BaseBox
+      height="95px"
+      width="125px"
+      display="inline-flex"
+      flexDirection="column"
+      alignItems="center"
+      gap="spacing.6"
+    >
+      <IconComponent {...args} />
+      <BaseBox
+        style={{
+          fontSize: 12,
+          width: '90%',
+          overflow: 'hidden',
+          textOverflow: 'ellipsis',
+          textAlign: 'center',
+        }}
+      >
+        {iconName}
+      </BaseBox>
+    </BaseBox>
+  );
+};
+
 export const AllIcons: ComponentStory<ComponentType<IconProps>> = ({ ...args }) => {
   return (
     <BaseBox>
-      {Object.keys(iconMap).map((icon, key) => {
-        const IconComponent = iconMap[icon];
-        return (
-          <BaseBox
-            height="95px"
-            width="125px"
-            display="inline-flex"
-            flexDirection="column"
-            alignItems="center"
-            gap="spacing.6"
-            key={key}
-          >
-            <IconComponent {...args} />
-            <BaseBox
-              style={{
-                fontSize: 12,
-                width: '90%',
-                overflow: 'hidden',
-                textOverflow: 'ellipsis',
-                textAlign: 'center',
-              }}
-            >
-              {icon}
-            </BaseBox>
-          </BaseBox>
-        );
-      })}
+      {isChromatic() ? (
```

Comment:
rendering only one icon in chromatic

Reply:
<img width="1637" alt="Screenshot 2023-07-28 at 4 22 09 PM" src="https://github.com/razorpay/blade/assets/11384858/75eae33b-d3ae-413e-ad36-0f808e32fd9b">

Reply:
Should we? because for chromatic it should not matter component ke andar if we are rendering 1 or 1000 subcomponents right? the chromatic will only take 1 snapshot either way.

Reply:
hmm true 🤔